lib/vtls/rustls.c

```diff
@@ -589,6 +602,21 @@ cr_connect_nonblocking(struct Curl_cfilter *cf,
   DEBUGASSERT(false);
 }
 
+static CURLcode
+cr_connect_nonblocking(struct Curl_cfilter *cf,
+                       struct Curl_easy *data, bool *done)
+{
+  cr_connect_common(cf, data, done, false);
+}
+
+static CURLcode
+cr_connect_blocking(struct Curl_cfilter *cf UNUSED_PARAM,
+           struct Curl_easy *data UNUSED_PARAM)
+{
+  bool done; // ignored
```

Comment:
~~~
vtls/rustls.c:616:14: error: C++ style comments are not allowed in ISO C90
  616 |   bool done; // ignored
      |              ^
vtls/rustls.c:616:14: note: (this will be reported only once per input file)

Reply:
(it would be cool if checksrc could do this kind of comment itself...)